packages/core/package.json

```diff
@@ -70,6 +70,8 @@
     "eslint": "8.19.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-simple-import-sort": "7.0.0",
+    "eslint-plugin-unused-imports": "2.0.0",
```

Comment:
sorta get frustrated with this plugin cause it sometimes deletes the imports i added if i save before using them :shrug: i'm fine keeping it, cause i can usually undo. just a bit on the fence about it since i save a lot